gulp/Gulpfile.js

```diff
@@ -0,0 +1,64 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  Declare the packages that will be required and assign each one to a variable
+//
+////////////////////////////////////////////////////////////////////////////////
+
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+
+
```

Comment:
It's for readability in large files. Perhaps since this file may not get too huge we don't need it? It's just habit for me these days but I certainly don't mind taking it out if you feel its unnecessary.